spring-cloud-azure-autoconfigure/pom.xml

```diff
@@ -73,6 +77,34 @@
             <optional>true</optional>
         </dependency>
 
+        <!-- Application Insights -->
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>applicationinsights-core</artifactId>
+            <version>${azure.applicationinsights.version}</version>
+        </dependency>
+
     </dependencies>
 
-</project>
\ No newline at end of file
+    <build>
+        <resources>
+            <!-- Filtering Resource -->
+            <resource>
+                <directory>src/main/resources</directory>
```

Comment:
This filter means not including this file in building phase, right? Then why need this file?

Reply:
No, it will include the file and replace the `placeholder`.